source/content/guides/multisite/01-introduction.md

```diff
@@ -41,17 +41,19 @@ We do not support uses of WordPress Multisite that run functionally-different or
 ## Request a WordPress Multisite
 WordPress Multisite requires a special configuration that is only available to select customers. Access to WordPress Multisite requires a Gold, Platinum, or Diamond level [Workspace plan](https://pantheon.io/plans/pricing).
 
-By default, Pantheon employees must create new Multisites on your behalf. Reach out to your account manager to request site creation for a new WordPress Multisite.
+By default, Pantheon employees must create the WordPress Multisite upstream on your behalf so your team can create new Multisites on the platform. This is because they use the `wordpress_network` [framework](/glossary/#framework) which includes Multisite specific Nginx configurations and differs from the default `wordpress` framework. Reach out to your account manager to request site creation for a new WordPress Multisite.
 
 If you want to create new WordPress Multisites self-serve and on-demand in the Pantheon Dashboard, reach out to your account manager and request a [Custom Upstream](/guides/custom-upstream) for your workspace based on Pantheon's WordPress Multisite framework.
 
-Existing WordPress sites cannot be converted to a multisite, however they can be [migrated](/migrate-wordpress-multisite).
+Existing WordPress sites cannot be converted to WordPress Multisites (since they are different frameworks), however you can spin up a new WordPress Multisite and then [manually migrate](/migrate-manual) the code, database and files from your existing site. 
 
 If you don't have an account manager, you can [contact sales](https://pantheon.io/contact-us).
```

Comment:
```suggestion
```

This feels out of place, relocated it to the end of line 44 in my suggestion above